jobs/pacman-awsrules/src/main/java/com/tmobile/cloud/redhatpolicies/misc/RedhatNotScanningClusterPolicy.java

```diff
@@ -0,0 +1,100 @@
+
+    @Override
+    public PolicyResult execute(Map<String, String> policyParam, Map<String, String> resourceAttributes) {
+        logger.debug("RedhatNotScanningClusterPolicy started");
+        if (MapUtils.isEmpty(policyParam) || MapUtils.isEmpty(resourceAttributes) || (MapUtils.isNotEmpty(policyParam)
+                && !PacmanUtils.doesAllHaveValue(policyParam.get(PacmanRuleConstants.SEVERITY),
+                policyParam.get(PacmanRuleConstants.CATEGORY)))) {
+            logger.info(PacmanRuleConstants.MISSING_CONFIGURATION);
+            throw new InvalidInputException(PacmanRuleConstants.MISSING_CONFIGURATION);
+        }
+        String policyId = policyParam.get(PacmanSdkConstants.POLICY_ID);
+        String severity = policyParam.get(PacmanRuleConstants.SEVERITY);
+        String category = policyParam.get(PacmanRuleConstants.CATEGORY);
+        MDC.put("executionId", policyParam.get("executionId"));
+        MDC.put("policyId", policyId);
+        try {
+            String pacmanHost = PacmanUtils.getPacmanHost(PacmanRuleConstants.ES_URI);
+            String clusterName = resourceAttributes.get("name");
+            String region = resourceAttributes.get("region");
+            String accountId = resourceAttributes.get(PacmanRuleConstants.ACCOUNTID);
+            Map<String, Object> mustFilter = new HashMap<>();
+            mustFilter.put("status.providerMetadata.google.clusterName", clusterName);
+            mustFilter.put("status.providerMetadata.zone", region);
+            mustFilter.put("status.providerMetadata.google.project", accountId);
+            HashMultimap<String, Object> shouldFilter = HashMultimap.create();
+            Map<String, Object> mustTermsFilter = new HashMap<>();
+            String endPoint = pacmanHost + REDHAT_CLUSTER_URL;
+            Set<String> resourceName = PacmanUtils.getValueFromElasticSearchAsSet(endPoint, mustFilter, shouldFilter,
+                    mustTermsFilter, "_resourcename", null);
+            if (resourceName == null || resourceName.isEmpty()) {
+                List<LinkedHashMap<String, Object>> issueList = new ArrayList<>();
+                LinkedHashMap<String, Object> issue = new LinkedHashMap<>();
+                Annotation annotation = Annotation.buildAnnotation(policyParam, Annotation.Type.ISSUE);
+                annotation.put(PacmanSdkConstants.DESCRIPTION, "Cluster - " + clusterName + " not scanned by redhat acs found");
+                annotation.put(PacmanRuleConstants.SEVERITY, severity);
+                annotation.put(PacmanRuleConstants.CATEGORY, category);
+                issue.put(PacmanRuleConstants.VIOLATION_REASON, "Cluster - " + clusterName + " not scanned by redhat acs found");
+                issueList.add(issue);
+                annotation.put("issueDetails", issueList.toString());
+                logger.debug("========RedhatNotScanningClusterPolicy ended with annotation {} : =========", annotation);
```

Comment:
do not use repetitive symbols in logs. keep logs clean, short and simple

Reply:
Sure, removed